src/pages/dashboardGroupEdit/components/GroupForm.tsx

```diff
@@ -293,12 +293,12 @@ export default function GroupForm({
   });
 
   useEffect(() => {
-    if (formik.dirty) {
+    if (formik.dirty || productSelected) {
```

Comment:
is this way the popup showing up also when a group is opened in edit mode, but without changing anything?